common/theme.c

```diff
@@ -123,3 +134,48 @@ void themeStartup(ThemePreset preset) {
     }
     config_destroy(&cfg);
 }
+
+void GetThemePathFromConfig(const char* themePath) {
+    config_t cfg = {0};
+    config_setting_t *setting;
+    config_init(&cfg);
+
+    char tmp_path[PATH_MAX] = {0};
+
+    #ifdef __SWITCH__
+    tmp_path[0] = '/';
+    #endif
+
+    strncat(tmp_path, "config/nx-hbmenu/setting.cfg", sizeof(tmp_path)-2);
+    bool good_cfg = config_read_file(&cfg, tmp_path);
+    
+    if(good_cfg) {
+        setting = config_lookup(&cfg, "themePath");
+        config_setting_lookup_string(setting, "themePath", &themePath);
```

Comment:
, themePath

Reply:
? are you refering to the &? it requires a const char **, hence the & of a const char *